stdlib/3/asyncio/base_subprocess.pyi

```diff
@@ -0,0 +1,59 @@
+
+from typing import Optional, Union, Sequence, IO, Any, List, Deque, Dict, Tuple, Callable
+import subprocess
+
+from . import events
+from . import protocols
+from . import futures
+from . import transports
+
+_File = Optional[Union[int, IO[Any]]]
+
+class BaseSubprocessTransport:
```

Comment:
The implementation derives from `transports.SubprocessTransport`.

Reply:
Update to match the implementation